prometheus/base/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
```

Comment:
please add to the top of the file:

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
```

In addition, you need to define the namespace for the resources.

```
namespace: o11y
```

and add another resource for the namespace definition

```
apiVersion: v1
kind: Namespace
metadata:
  name: o11y
spec: {}
```

Reply:
Thanks! updated